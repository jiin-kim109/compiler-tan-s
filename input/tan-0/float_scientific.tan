main {
    var r := 3.14e+4;
    print r \n;
    r := 3.14e-4;
    print r \n;
}