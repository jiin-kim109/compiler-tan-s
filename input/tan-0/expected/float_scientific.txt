31400.000000
0.000314
